### ISSUE TITLE
please clarify definition of <header> element

### ISSUE THREAD

**Issue by prlbr** [w3c/html#1405]
I have some trouble interpreting the following sentence from https://w3c.github.io/html/sections.html#the-header-element:

> The `<header>` element represents introductory content for its nearest ancestor `<main>` element or sectioning content or sectioning root element. A `<header>` typically contains a group of introductory or navigational aids.

Consider this example:

```html
<main>
  <h1>Tomato History</h1>
  <p>
    A tomato is an edible fruit, but by many considered a vegetable.
    It has a long and interesting history.
  </p>
  <h2>Roots</h2>
  <p>…</p>
  <p>…</p>
  <header>
    <h2>An aztecs’ favorite</h2>
    <p>1300–1500 AD</p>
  </header>
  <p>…</p>
  <p>…</p>
  <header>
    <h2>The road to world fame</h2>
    <p>1500–1700 AD</p>
  </header>
  <p>…</p>
  <p>…</p>
  …
```

It feels like the `<header>` elements in these examples should simply group the `<h2>` headings with the date byline below – this kind of use is also recommended in https://w3c.github.io/html/common-idioms-without-dedicated-elements.html#subheadings-subtitles-alternative-titles-and-taglines .

But the definition quoted above seems to say that all the `<header>`s would represent introductory content for the `<main>` element, which is the nearest ancestor `<main>` element or [sectioning content](https://w3c.github.io/html/dom.html#sectioning-content) or [sectioning root](https://w3c.github.io/html/sections.html#sectioning-roots) element.

Even if we wouldn’t just consider explicit sections, but also implicit sections, the defintion would still seem to associate the `<header>` elements unintuitevly as introductory content of the section that is started by the heading before it and not by the heading inside it.

It seems to me that with the current definition, `<header>` could only be used sensibly with explicit sectioning elements, but not without them as in my example. I suppose that this is not what is intended.

The [WHATWG definition](https://html.spec.whatwg.org/multipage/sections.html#the-header-element) does not have that problem as it simply defines …

> The `header` element represents a group of introductory or navigational aids.

… without further details. I don’t think that this is the exact way to go – it is good that the full W3C definition includes the implications for assistive technology. But I hope that the definition can be simplified to be more intuitive and applicable.

**Comment by torressam333** [comment]
So there can be a header element in more than one place in an html file? Must it remain in the <head> element?

**Comment by Alohci** [comment]
@prlbr. It seems that in the examples given, the header element is being used as a grouping for the introductory information of the implicit sub-section created by the h2 element. But there is no requirement that a header element contain an h1-h6 element in which case, what is the content that is being introduced? Should a header element without a contained h1-h6 element imply a sub-section creation the way h1-h6 elements do? Or should a header element without a contained h1-h6 element be invalid? As far as I can see, neither the W3C nor the Living spec provide an example of a header element that does not contain a h1-h6 element.

**Comment by torressam333** [comment]
Thank you for the excellent explanation. This is very useful!
